tests/nnstreamer_filter_tensorflow_lite/runTest.sh

```diff
@@ -67,6 +67,7 @@ fi
 PATH_TO_MODEL="../test_models/models/mobilenet_v1_1.0_224_quant.tflite"
 PATH_TO_LABEL="../test_models/labels/labels.txt"
 PATH_TO_IMAGE="../test_models/data/orange.png"
+SYSTEM=`uname -m`
```

Comment:
In the case of Tizen/ARM, "**uname -m**" command will display "x86_64" without "armv7l or aarch64".
Does the statement handle this case?

Reply:
```bash
invain@mymate:/proc/sys/fs/binfmt_misc$ gbs chroot ~/GBS-ROOT-SNAPSHOT/local/BUILD-ROOTS/scratch.armv7l.0/
info: chroot /home/invain/GBS-ROOT-SNAPSHOT/local/BUILD-ROOTS/scratch.armv7l.0
bash-3.2$ uname -m
armv7l

```

It's okay. No problem.